### PR TITLE
Fix #116: Show 'heads(draft())' and 'max(public())' revisions in "Hg: Update to..."

### LINKED ISSUE #116: Show 'heads(draft())' and 'max(public())' revisions in "Hg: Update to..."
Mercurial does not force you to create bookmarks or branches when committing. To improve usability, we could display all draft heads in the Hg: Update to... command.

### PR DESCRIPTION
**Pull request by hdpoliveira**
Demo:
update draft heads

### CODE REVIEW

File: src/hg.ts [hunk 1]
```diff
@@ -1534,13 +1534,13 @@ export class Repository {
         return activeBookmark;
     }
 
-    async getLogEntries({
+    async getNativeCommits({
```
Comment (incidentist) [inline src/hg.ts:1537]: This should be private, I think.

File: src/commands.ts [hunk 2]
```diff
@@ -1048,7 +1048,14 @@ export class CommandCenter {
...
-            refs = await repository.getRefs();
+
+            const [branches, tags, maxPublic, draftHeads] = await Promise.all([
+                repository.getBranches(),
+                repository.getTags(),
+                repository.getPublicTip(),
+                repository.getDraftHeads(),
+            ]);
+            refs = [...branches, ...tags, ...maxPublic, ...draftHeads];
         }
```
Comment (incidentist) [inline src/commands.ts:1059]: Tags, public tip and draft heads are only included when the repo uses branches. They aren't included if it uses bookmarks. Why not?

Comment (hdpoliveira) [reply src/commands.ts:1059]: I don't really know how the development using bookmarks goes. I'm guessing it is just like git: you want to start a new thing, first thing you do is create a bookmark. So there wouldn't be any heads without bookmarks, so it does not make much sense to add this to the bookmarks code.

File: src/hg.ts (hunk 1 above)
Comment (hdpoliveira) [reply src/hg.ts:1537]: Definitely

File: src/commands.ts (hunk 2 above)
Comment (incidentist) [reply src/commands.ts:1059]: It's possible to use bookmarks and still have heads without bookmarks, but it's less useful than when using branches. However, if someone *does* have bookmark-less heads, it feels like those heads should show up in the list. Using bookmarks also doesn't stop someone from wanting to update to a specific tag or the public tip.

**Comment by hdpoliveira** [comment]
@incidentist all good now :)

**Pull request merged**